### DISCUSSION TITLE
BoX pop app MatchClick Callback

### DISCUSSION THREAD

**Discussion by scroodge** [Drarig29/brackets-viewer.js#109] [Q&A] [answered]
Hi, it looks i can't find how to call  MatchClick Callback inside BoX pop-up. Can someone sugest how to do this?

**Comment by Drarig29** [comment]
Hi @scroodge, what did you try?

Did you look at the demos?

https://github.com/Drarig29/brackets-viewer.js/blob/5bdcdaa16b285694252412e6ad337bf630d08738/demo/with-api.html#L40-L67

**Reply by scroodge** [reply]
yes sure! for main window it perfectly works  - what about this pop_up?
<img width="524" alt="Screenshot 2024-06-07 at 22 08 13" src="https://github.com/Drarig29/brackets-viewer.js/assets/1701571/8dad711a-684f-4966-94e5-b6dd7c383ccb"> WB Semi 1 - there two matches and i want to click on it and have call back to enter result. What am i missing? Thanks in advance!

**Reply by Drarig29** [reply, marked as answer]
You can take inspiration from [this UI demo](https://github.com/Drarig29/brackets-viewer.js/blob/master/demo/with-ui.html), which uses [part of this code](https://github.com/Drarig29/brackets-viewer.js/blob/master/src/form.ts).

But sorry, I won't provide you with more help than this because it's the consumer part of the library.
